### PR TITLE
feat: user friendly network name

### PR DESCRIPTION
**Pull request by j0hnfl0w**
Closes: https://github.com/snapshot-labs/sx-ui/issues/475

### CODE REVIEW

File: src/components/S/ISelect.vue [hunk 1]
```diff
@@ -39,8 +39,12 @@ const inputValue = computed({
...
     <select v-model="inputValue" class="s-input">
       <option disabled value="">Please select one</option>
-      <option v-for="option in definition.enum" :key="option" :value="option">
-        {{ option }}
+      <option
+        v-for="option in definition.options || definition.enum"
+        :key="option"
+        :value="option?.id || option"
+      >
+        {{ option?.name || option }}
```
Comment (Sekhmet) [inline src/components/S/ISelect.vue:47]: Do we expect `option` to be undefined?

Comment (j0hnfl0w) [reply src/components/S/ISelect.vue:47]: No it's either a string or an object {id,name}

Comment (Sekhmet) [reply src/components/S/ISelect.vue:47]: We don't need `?.` then?

Comment (j0hnfl0w) [reply src/components/S/ISelect.vue:47]: yep

**Review by Sekhmet — APPROVED** [review]
tACK

**Pull request merged**